benchmarks/inference-server/text-generation-inference/custom-metrics-stackdriver-adapter/README.md

```diff
@@ -0,0 +1,31 @@
+# Custom Metrics Stackdriver Adapter
```

Comment:
No action here, just an interesting possibility for potential work for the custom metrics stackdriver adapter modules

We've been utilizing infrastructure modules from Cloud Foundry (eg. example for GKE Hub, https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/tree/master/modules/gke-hub) when possible for getting default GKE configurations and easier TF automation.

Wonder if adding some modules there for GKE HPA at some point would also be a great place for customers to find these generic modules for configuration.